FrontEnd/FrontEnd.Web/Services/RecipeService.cs

```diff
@@ -36,4 +38,15 @@ public async Task<bool> DeleteByIdAsync(int id)
         var response = await _httpClient.DeleteAsync($"recipes/{id}");
         return response.IsSuccessStatusCode;
     }
+
+    public async Task<Recipe> CreateAsync(RecipeCreateDTO request)
+    {
+        var response = await _httpClient.PostAsJsonAsync("recipes", request);
+        return await response.Content.ReadFromJsonAsync<Recipe>() ?? throw new InvalidOperationException();
+    }
+
+    public Task UpdateAsync(int id, RecipeUpdateDTO request)
```

Comment:
Any reason why this method is not async and doesn't await like all the others? You seem to call an async method.

Reply:
I guess it's debatable, practically I don't think it makes much of a difference. Returning a task directly has less overhead, but there's other tradeoffs :D I usually return a task when that's the single call